cmd/protoc-gen-micro/examples/greeter/greeter.proto

```diff
@@ -15,6 +15,7 @@ service Greeter {
 
 message Request {
 	string name = 1;
+	optional string msg = 2;
 }
```

Comment:
Should this be optional msg in response?

Reply:
It's both fine to be optional in request and response, but an optional response maybe not an good design I think.
Cause it's just used for testing, I can change to optional msg in response if you need.